tools/c7n_org/c7n_org/cli.py

```diff
@@ -171,14 +171,15 @@ def report_account(account, region, policies_config, output_path, debug):
 @click.option('-s', '--output-dir', required=True, type=click.Path())
 @click.option('-a', '--accounts', multiple=True, default=None)
 @click.option('--field', multiple=True)
+@click.option('--no_default_fields', default=False, is_flag=True)
```

Comment:
generally try to avoid `_` in cli flags vs `-`

Reply:
oh ok I will get that changed, thought the - was messing up the variable recognition but must have just been the way it was displayed.